rook-ceph.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest. The name defines how
+  # the plugin is invoked, for example: `kubectl rook-ceph`
+  name: rook-ceph
+spec:
+  # 'version' is a valid semantic version string (see semver.org)
+  version: "v0.0.1"
+  # 'homepage' usually links to the GitHub repository of the plugin
+  homepage: https://github.com/rook/kubectl-rook-ceph
+  # 'shortDescription' explains what the plugin does in only a few words
+  shortDescription: "rook-ceph kubectl plugin"
+  description: |
+    Krew plugin to run kubectl commands with rook-ceph.
+  # 'platforms' specify installation methods for various platforms (os/arch)
+  platforms:
+  - selector:
+      matchExpressions:
+      - key: "os"
+        operator: "In"
+        values:
+        - darwin
```

Comment:
windows should work too right?

Reply:
I have no ideas about the window :grin:

Reply:
we could add an integration test with a macos/windows runner to test if possible. An item for the later though.

Reply:
Right, hopefully the plugin works wherever kubectl works.

Reply:
Let's add window anyway, I don't see any reasons why it wouldn't work.